laravel/advanced-guides/global-sqlite-litefs.html.md.erb

```diff
@@ -0,0 +1,307 @@
+---
+title: Global SQLite with LiteFS
+layout: framework_docs
+objective: Go LiteFS-global with your SQLite-backed Laravel Fly App!
+---
+
+So you've decided to give SQLite a chance huh? If your answer's a bold, brave <b><i>yes</i></b>, then welcome! There's priceless <b>walkthough treasures</b> waiting for your read below! 
```

Comment:
```diff
- waiting for your read below
+ waiting for you to read below
```

(Or maybe something like `waiting for your viewing pleasure below`)

Reply:
right-o! I need to up my skills on clear writing, thanks @fideloper!

Reply:
fixed!